tests/modules/images/webp-uploads/webp-uploads-test.php

```diff
@@ -28,6 +33,7 @@ function test_webp_uploads_applies_filter_based_on_system_support() {
 	function test_webp_uploads_filter_image_editor_output_format_with_support() {
 		// Mock a system that supports WebP.
 		add_filter( 'wp_image_editors', array( $this, 'mock_wp_image_editor_supports' ) );
+		add_filter( 'image_editor_output_formats', array( $this, 'return_webp_array' ) );
```

Comment:
See above, why does this need to be here?

Reply:
removing